ui/library/src/context/ScrollContext.ts

```diff
@@ -216,6 +223,21 @@ export function useScrollContextProps(): IScrollContext {
     };
   }, [scrollRoot, observerIndex]);
 
+  const recalibrateScrollPosition = React.useCallback(
```

Comment:
@ericmarsh995 just a nit but put a comment to explain that this will just calculate a new position when u zoom in or out so people won't forget